R/global.R

```diff
@@ -23,14 +23,20 @@
 # from another package, they need to be added to this package's NAMESPACE.
 
 utils::globalVariables(
-  c(
-    ".",
+  c(".",
+    "analysis",
+    "asy_event",
+    "asy_n",
+    "asy_time",
+    "bound",
     "ex1_delayed_effect",
-    "N",
     "cte",
+    "cross_lower",
+    "cross_upper",
     "dropout_rate",
     "dropout_time",
     "duration",
+    "ends_with",
```

Comment:
`ends_with()` is a function from tidyselect, not a non-standard evaluation variable name. So `#' @importFrom tidyselect ends_with` in the roxygen documentation is the right solution.

Reply:
Thanks for pointing it out, @nanxstats ! I deleted the `ends_with` in the latest commit.

Reply:
Perhaps I didn't point it out clearly enough - you need to import it to avoid check notes.